test/util/platform_unit.js

```diff
@@ -30,6 +30,12 @@ describe('Platform', () => {
 
   // cspell: disable-next-line
   const vizio = 'Mozilla/5.0 (X11; Linux aarch64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.141 Safari/537.36 CrKey/1.0.999999 VIZIO SmartCast(Conjure/MTKF-5.1.516.1 FW/0.6.11.1-2 Model/V50C6-J09)';
+  // cspell: disable-next-line
+  const chromecastBuiltinOrOlder = 'Mozilla/5.0 (X11; Linuz armv71) AppleWbKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.225 Safari/537.36 CrKey/1.56.500000';
```

Comment:
Linuz? AppleWbKit? Did you take this string from actual device? Looks malformed at a first glance.